src/nav/components.yml

```diff
@@ -52,6 +52,8 @@
           title: Checkbox
         - id: /components/input
           title: Input
+        - id: /components/multithumbrange
+          title: Multi-Thumb Range
```

Comment:
For hyphenated values do we normally `uppercase` the second half?

Reply:
Great call. Will update `range` PR link as well.

Reply:
Oh wait, that's funny, I read something different the first time through which is that maybe this should be `multithumb-range` because the space previously has been a dash (i.e. `components/button-group`). 

I was following [W3s capitilization lead for this one](https://www.w3.org/TR/wai-aria-practices-1.1/examples/slider/multithumb-slider.html). What do you think? Is `Multi-thumb Range` our alt consideration?

Reply:
Would be nice if our https://garden.zendesk.com/content/punctuation#hyphens-and-dashes content had the answer. @vsrixyz?

Reply:
This should be `Multi-thumb range.` The first part of the expression is a single word and would follow our sentence case standard.